ci/build-gmt-master.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+#
+# Build the lastest GMT source codes
+#
```

Comment:
Would be good to list all the environment variables that can be set here so that it's easy to find.